eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Epoch.hs

```diff
@@ -242,13 +243,14 @@ epochTransition = do
         , reDRepState = vstate ^. vsDRepsL
         }
     ratSig = RatifySignal . snapshotActions . prevGovSnapshots $ cgGovSnapshots govSt
+    futureEnactState = govSt ^. cgFutureEnactStateL
```

Comment:
As we said in the meeting, `enactState` would be a less confusing name than `futureEnactState`